### ISSUE TITLE
Need to use newer release of mimemagic

### ISSUE THREAD

**Issue by AkihiroKitada** [pivotal/postfacto#351]
According to [Gemfile.lock](https://github.com/pivotal/postfacto/blob/master/api/Gemfile.lock#L166), mimemagic 0.3.5 is applied with API part.

However, according to [rubygems.org](https://rubygems.org/gems/mimemagic/versions), mimemagic 0.3.5 has been yanked and we can't download the release any more. As a result, it fails in the compile phase when trying to deploy on Tanzu Application Service with the following error:
```
    :
   -----> Installing dependencies using bundler 2.0.1
          Running: bundle install --without development:test --jobs=4 --retry=4 --path /tmp/contents322398787/deps/0/vendor_bundle --binstubs /tmp/contents322398787/deps/0/binstubs --deployment
          Fetching gem metadata from https://rubygems.org/............
          Your bundle is locked to mimemagic (0.3.5), but that version could not be found
          in any of the sources listed in your Gemfile. If you haven't changed sources,
          that means the author of mimemagic (0.3.5) has removed it. You'll need to update
          your bundle to a version other than mimemagic (0.3.5) that hasn't been removed
          in order to install.
          **ERROR** Unable to install gems: exit status 7
   Failed to compile droplet: Failed to run all supply scripts: exit status 15
   Exit status 223
    :
BuildpackCompileFailed - App staging failed in the buildpack compile phase
```

Could you apply newer version of mimemagic module?

**Comment by dwettstein** [comment]
Updating to Rails Version 6.0.3.6 is probably sufficient, as this version of Rails removes the dependency to _mimemagic_.

See also: https://github.com/rails/rails/issues/41750

**Comment by textbook** [comment]
Fixed in v4.3.1
